scripts/postlink/ios/postlink.js

```diff
@@ -4,8 +4,20 @@ var inquirer = require('inquirer');
 var path = require("path");
 var plist = require("plist");
 
+// Fix for https://github.com/Microsoft/react-native-code-push/issues/477
+// Try to resolve package.json, assuming we are working off of the project root
+// We will use this later to fetch the correct AppDelegate and Plist file, in case there are multiple.
+var package = require(path.resolve('./package.json'));
```

Comment:
Ohh, that sounds plausible. I guess maybe `require("./package.json")` resolves relative to `__dirname` (the folder the file is located in), and `path.resolve("./package.json")` resolves relative to the user's pwd.  This makes sense to me!